test_outputs/connlist/test1_anp_conn_from_parsed_res.txt

```diff
@@ -0,0 +1,20 @@
+0.0.0.0-255.255.255.255 => x/a[Pod] : All Connections
+0.0.0.0-255.255.255.255 => x/b[Pod] : All Connections
+0.0.0.0-255.255.255.255 => y/a[Pod] : All Connections
+0.0.0.0-255.255.255.255 => y/b[Pod] : All Connections
+x/a[Pod] => 0.0.0.0-255.255.255.255 : All Connections
+x/a[Pod] => x/b[Pod] : All but: TCP 80
```

Comment:
@shireenf-ibm , the change of the connection string to "all but" should be in a separate branch and separate issue, not in the base branch (to be decided if/when to be used).

Reply:
done (reverted from base PR and moved to a new PR #395)